examples/user_guide/Custom_Components.ipynb

```diff
@@ -349,7 +359,7 @@
     "    _template = \"\"\"\n",
     "    <select id=\"select\" value=\"${value}\" style=\"width: ${model.width}px\">\n",
     "      {% for option in options %}\n",
-    "      <option id=\"option\">${option}</option>\n",
+    "      <option id=\"option-{{ loop.index0 }}\">{{ option }}</option>\n",
```

Comment:
You can keep the existing example. But it is misleading as you will not use that syntax in practice to create a Select element for literal values as the example shows. That the example is misleading takes a long time to figure out as you *trust* the documentation.

Reply:
It's not necessarily misleading I guess just a little silly to render some text by creating an entire child model.